README.md

```diff
@@ -1,8 +1,31 @@
 ## Passage Licensing Runtime  
 
-Passage Licensing Runtime layer includes interfaces and its implementation for Eclipse Equinox
+The Passage Licensing Runtime is a part of the Eclipse Passage licensing system. 
```

Comment:
"is a part of the Eclipse Passage licensing system."
not yet, lets reformulate to
"is a part of the Passage Licensing Solution. Please see "Eclipse Passage" project proposals at https://projects.eclipse.org/proposals/eclipse-passage"